src/core/src/energy/global/wolf.rs

```diff
@@ -172,10 +174,13 @@ impl GlobalPotential for Wolf {
 
     fn energy(&self, system: &System) -> f64 {
         let natoms = system.size();
-        let mut res = 0.0;
-        for i in 0..natoms {
+
+        (0..natoms).into_par_iter().map(|i|{
```

Comment:
That indeed happens often. Maybe in the same idea as `Zip` in `ndarray_parallel`, have a `par_map` shortcut that does this.
We would have `(0..nations).par_map(|i| ...)`.